src/services/completions.ts

```diff
@@ -964,11 +964,11 @@ namespace ts.Completions {
                     const moduleSymbol = firstAccessibleSymbol.parent;
                     symbolToOriginInfoMap[getSymbolId(firstAccessibleSymbol)] =
                         !moduleSymbol || !isExternalModuleSymbol(moduleSymbol) ? { kind: SymbolOriginInfoKind.SymbolMemberNoExport } : { kind: SymbolOriginInfoKind.SymbolMemberExport, moduleSymbol, isDefaultExport: false };
+                    return;
                 }

```

Comment:
I think in the else part you want to check `includeInsertTextCompletions` to be true and only then add symbol

Reply:
@samelhusseini do you mean `preferences.includeInsertTextCompletions`?

```typescript
if (preferences.includeCompletionsWithInsertText) {
   symbols.push(symbol);
}
```

Reply:
Yes

Reply:
@sheetalkamat oke, thanks 👍

Reply:
@sheetalkamat Could you review this PR again? Thanks in advance.